lib/linguist/heuristics.yml

```diff
@@ -440,6 +440,10 @@ disambiguations:
     pattern: '(?i:\^(this|super)\.|^\s*~\w+\s*=\.)'
   - language: Scala
     pattern: '(^\s*import (scala|java)\.|^\s*class\b)'
+- extensions: ['.sol']
+  rules:
+  - language: Solidity
+    pattern: 'pragma\s+solidity'
```

Comment:
**Nitpick:** I'd add word boundaries to this expression, so something like `not_a_pragma solidity1` won't match:

~~~diff
   rules:
   - language: Solidity
-     pattern: 'pragma\s+solidity'
+     pattern: '\bpragma\s+solidity\b'
~~~

Reply:
Makes sense!

Reply:
Done. 👍 I've pushed the changes to your branch, so this is good to merge, IMO.